certificate-service/app/src/main/java/se/inera/intyg/certificateservice/application/common/ActionEvaluationFactory.java

```diff
@@ -76,6 +77,7 @@ public ActionEvaluation create(PatientDTO patient, UserDTO user, UnitDTO unit, U
                         .map(HealthCareProfessionalLicence::new)
                         .toList()
                 )
+                .responsibleHospName(new ResponsibleHospName(user.getResponsibleHospName()))
```

Comment:
Do we want to use "responsibleHospName" or name it something different? I remember we removed the term hosp when working with intyg-proxy.

Reply:
Maybe we should update it to ResponsibleIssuer?